js/snippetPreview.js

```diff
@@ -397,3 +672,141 @@ YoastSEO.SnippetPreview.prototype.setFocus = function( ev ) {
 		}
 	}
 };
+
+/**
+ * Binds the reloadSnippetText function to the blur of the snippet inputs.
+ */
+SnippetPreview.prototype.bindEvents = function() {
+	var targetElement,
+		elems = [ "title", "slug", "meta-description" ],
+		focusBindings = [
+			{
+				"click": "title",
+				"focus": "title"
+			},
+			{
+				"click": "urlPath",
+				"focus": "urlPath"
+			},
+			{
+				"click": "urlBase",
+				"focus": "urlPath"
+			},
+			{
+				"click": "metaDesc",
+				"focus": "metaDesc"
+			}
+		];
+
+	for ( var i = 0; i < elems.length; i++ ) {
+		targetElement = document.getElementsByClassName( "js-snippet-editor-" + elems[ i ] );
+		targetElement = targetElement[0];
```

Comment:
Why not just immediately call `targetElement = document.getElementsByClassName( "js-snippet-editor-" + elems[ i ] )[0];`?